app/controllers/application_controller.rb

```diff
@@ -10,11 +10,27 @@ class NotFoundException < RuntimeError
   end
 
   before_action :doorkeeper_authorize!
+  before_action :update_log
 
   private
 
   # Doorkeeper methods
   def current_resource_owner
     User.find(doorkeeper_token.resource_owner_id) if doorkeeper_token
   end
+
+  # Function to update the audit log as new events occur in a user's account.
+  def update_log
+    current_user = current_resource_owner
+    if current_user.nil?
+      current_user = "N/A"
+    else
+      current_user = current_user.id
+    end
+    description = controller_name + " " + action_name
+    params.each do |variable|
+      description = description + " " + variable.to_s
+    end
+    new_audit_log = AuditLog.create(requester_info: current_user, event: "event", event_time: (Time.new).inspect, description:description)
```

Comment:
The `event` column should have a meaningful value. If we don't have anything to put in there we should delete the column.

Reply:
@rdingwell and I discussed that and he said for now, "event" is a placeholder which we'll look to fill eventually.